configureWorkspace/configure_python.ts

```diff
@@ -32,6 +32,7 @@ WORKDIR /app
 ADD . /app
 
 # Using pip:
+RUN if [ ! -f requirements.txt ]; then pip3 freeze > requirements.txt; fi
```

Comment:
When I try to run the built image, I get this:

> docker run --rm -it -p 3000:3000/tcp hello-world:latest
/usr/local/bin/python3: No module named hello-world